FreydCategoriesForCAP/gap/precompiled_categories/CategoryOfColumnsAsOppositeOfCategoryOfRowsOfCommutativeRingPrecompiled.gi

```diff
@@ -261,7 +262,8 @@ end
         
 ########
 function ( cat_1, a_1 )
-    return a_1;
+    return ObjectifyObjectForCAPWithAttributes( rec(
```

Comment:
Do we regard this as a regression?

Reply:
... the return value used to be the input `a_1`.

Reply:
It is a regression, but I don't think investing time into this makes sense because I can't imagine this being a bottleneck in general applications.